docs/install_guides/2022.1.1/installing-openvino-from-archive-linux.md

```diff
@@ -0,0 +1,174 @@
+
+With the OpenVINO™ 2022.1.1 release, you can download and use archive files to install OpenVINO Runtime. 
+
+You can also check the [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes) for more information on updates in this release.
+
+> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter can be installed via [pypi.org](https://pypi.org/project/openvino-dev/) only.
+
+## System Requirements
+
+@sphinxdirective
+.. tab:: Operating Systems
+
+  * Ubuntu 18.04 long-term support (LTS), 64-bit
+  * Ubuntu 20.04 long-term support (LTS), 64-bit
+
+  .. note::
+     Since the OpenVINO™ 2022.1 release, CentOS 7.6, 64-bit is not longer supported.
+
+.. tab:: Hardware
+
+  Optimized for these processors:
+
+  * 6th to 12th generation Intel® Core™ processors and Intel® Xeon® processors 
+  * 3rd generation Intel® Xeon® Scalable processor (formerly code named Cooper Lake)
+  * Intel® Xeon® Scalable processor (formerly Skylake and Cascade Lake)
+  * Intel Atom® processor with support for Intel® Streaming SIMD Extensions 4.1 (Intel® SSE4.1)
+  * Intel Pentium® processor N4200/5, N3350/5, or N3450/5 with Intel® HD Graphics
+  * Intel® Iris® Xe MAX Graphics
+  * Intel® Neural Compute Stick 2
+  * Intel® Vision Accelerator Design with Intel® Movidius™ VPUs
+
+.. tab:: Processor Notes
+
+  Processor graphics are not included in all processors. 
+  See `Product Specifications`_ for information about your processor.
+  
+  .. _Product Specifications: https://ark.intel.com/
+
+.. tab:: Software
+
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
+  * `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
+     * Note that OpenVINO is gradually stopping the support for Python 3.6. Python 3.7 - 3.9 are recommended. 
+
+@endsphinxdirective
+
+## Installing OpenVINO Runtime
+
+@sphinxdirective
+
+.. important::
+   Before you start your journey with installation of the Intel® Distribution of OpenVINO™ toolkit, we encourage you to check the :ref:`code samples <code samples>` in C, C++, Python and :ref:`notebook tutorials <notebook tutorials>`, so you could see all the amazing things that you can achieve with our tool.
+
+@endsphinxdirective
+
+### <a name="install-openvino"></a>Step 1: Download and Install the OpenVINO Package
+
+1. Select and download the OpenVINO™ archive files from [Intel® Distribution of OpenVINO™ toolkit download page](https://software.intel.com/en-us/openvino-toolkit/choose-download). There are typically two files for you to download: 
+   ```sh
+   l_openvino_toolkit_<operating system>_dev_<release version>_<package ID>.tgz
+   l_openvino_toolkit_<operating system>_dev_<release version>_<package ID>.tgz.sha256
```

Comment:
@artanokhov why packages are named with `_dev`? I suppose we agreed to remove such suffix to avoid confusions with tools (openvino-dev pypi package)

Reply:
Yes, correct - there is no '_dev_' affix anymore.
@xu-yuan1 please correct the names

Reply:
Hi @artanokhov , just to avoid any mistakes, can you directly make commit suggestions on the format of archive file? I will update the rest file names afterwards.